src/Columns/ColumnLowCardinality.cpp

```diff
@@ -742,8 +753,6 @@ void ColumnLowCardinality::Index::insertPositionsRange(const IColumn & column, U
         !insert_for_type(UInt64()))
         throw Exception("Invalid column for ColumnLowCardinality index. Expected UInt, got " + column.getName(),
                         ErrorCodes::ILLEGAL_COLUMN);
-
-    checkSizeOfType();
```

Comment:
checkSizeOfType() is not needed here, Since,  Line #745 make sure that  size is equal to size_of_type.